examples/dmrg/symmetric_dmrg.py

```diff
@@ -0,0 +1,179 @@
+"""
+Example of backend independent DMRG calculation for the
+spin 1/2 Heisenberg Model.
+"""
+
+from typing import Type, Text
+import tensornetwork as tn
+import numpy as np
+from tensornetwork import (FiniteMPO, FiniteMPS, U1Charge, Index,
+                           BlockSparseTensor, FiniteXXZ, FiniteDMRG)
+import jax
```

Comment:
Presumably we don't want to import this unless the Jax backend is desired? Otherwise this code will fail if Jax isn't installed.

Reply:
I think that's fine in the example folder. That's not really part of the package, and we're already importing this for example in `simple_mera.py`